### PR TITLE
Add  Running an Upsert

### PR DESCRIPTION
**Pull request by MichelDiz**
Adding doc for  Upserts.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/13)

### CODE REVIEW

**Review by martinmr — APPROVED** [review]
Some minor comments but it :lgtm: 

> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/13#-:-LoNS9HtE5PN8-LEUOtp:b2px8ns)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @MichelDiz)

___
*[README.md, line 234 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/13#-LoNRmBQ1KtVrnpiccVb:-LoNRmBQ1KtVrnpiccVc:b20w542) ([raw file](https://github.com/dgraph-io/dgraph-js-http/blob/ee195cce8fefc97b96f5a82693fdebc479014a28/README.md#L234)):*
> ```
> runs a mutation and also upsert transactions.
> ```
"runs mutations and upsert transactions"
___
*[README.md, line 234 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/13#-LoNS07SBY4FomHsBRxg:-LoNS07SBY4FomHsBRxh:bc4stqk) ([raw file](https://github.com/dgraph-io/dgraph-js-http/blob/ee195cce8fefc97b96f5a82693fdebc479014a28/README.md#L234)):*
> ```
> provides two main ways to set data: JSON and RDF N-Quad.
> ```

"provides two data formats" ...

**Review by martinmr — COMMENTED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/13)* status: all files reviewed, 2 unresolved discussions (waiting on @MichelDiz)

**Review by MichelDiz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/13)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @martinmr)

___
*[README.md, line 234 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/13#-LoNRmBQ1KtVrnpiccVb:-LoNUVgX1veiqihVaAM_:b-896fix) ([raw file](https://github.com/dgraph-io/dgraph-js-http/blob/ee195cce8fefc97b96f5a82693fdebc479014a28/README.md#L234)):*
<details><summary><i>Previously, martinmr (Martin Martinez Rivera) wrote…</i></summary><blockquote>

> ```
> runs a mutation and also upsert transactions.
> ```
"runs mutations and upsert transactions"
</blockquote></details>

Done.
___
*[README.md, line 234 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/13#-LoNS07SBY4FomHsBRxg:-LoNUW-lDCEwVPNPISZQ:b-896fix) ([raw file](https://github.com/dgraph-io/dgraph-js-http/blob/ee195cce8fefc97b96f5a82693fdebc479014a28/README.md#L234)):*
<details><summary><i>Previously, martinmr (Martin Martinez Rivera) wrote…</i></summary><blockquote>

> ```
> provides two main ways to set data: JSON and RDF N-Quad.
> ```

"provides two data formats" ...
</blockquote></details>

Done.

File: README.md [hunk 1]
```diff
@@ -229,6 +229,29 @@ Number of people named "Alice": 1
 Alice
 ```
 
+### Running an Upsert
+
+Txn#mutate(Mutation) runs mutations and upsert transactions. It takes in a Mutation object, which provides two data formats: JSON and RDF N-Quad. You can choose whichever way is convenient.
```
Comment (mangalaman93) [inline README.md:234]: This doesn't seem right. @prashant-shahi please have a discussion about this with me. Something seems off.

**Review by mangalaman93 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mangalaman93) [reply README.md:234]: Please DO NOT MERGE.

**Comment by mangalaman93** [comment]
@MichelDiz closing this PR. More work needs to be done in this client. I didn't realize that the interface of the http client is similar to the grpc client. @prashant-shahi will work on this.

**Pull request closed without merging**